src/util/logging.jl

```diff
@@ -26,7 +26,7 @@ is set."
 end
 @inline trace(msg...; kwargs...) = trace(STDERR, msg...; kwargs...)
 macro trace(args...)
-    TRACE && return esc(Expr(:call, :trace, args...))
+    TRACE && return esc(Expr(:call, :(CUDAdrv.trace), args...))
```

Comment:
Nice catch. Might be better to keep the `esc` fine grained though: leave the `trace` unescaped, and pass `$(map(esc, args)...)`